src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -120,29 +124,36 @@ function EpidemicModelProvider({ children }: EpidemicModelProviderProps) {
         // the first line is not the header row so we need to strip it
         rawCSV = rawCSV.substring(rawCSV.indexOf("\n") + 1);
 
-        const parsedArray = csvParse(rawCSV, (row) => {
-          const transformedRow = autoType(row);
-          // autotype won't catch formatted thousands and percentages,
-          // so run them through numeral
-          Object.entries(transformedRow).forEach(([key, value]) => {
-            if (value && typeof value === "string") {
-              const transformedValue = numeral(value).value();
-              transformedRow[key] =
-                transformedValue === null ? value : transformedValue;
-            }
-          });
-
-          return transformedRow;
-        })
+        const parsedArray = csvParse(
+          rawCSV,
+          (row): CountyLevelRecord => {
+            // we only need a few columns, which we will explicitly format
+            const totalIncarceratedPopulation = numeral(
+              row["Total Incarcerated Population"],
+            ).value();
+            const estimatedTotalCases =
+              (numeral(row.cases).value() || 0) * (1 / caseReportingRate);
+            return {
+              county: row.County || "",
+              state: row.State || "",
+              hospitalBeds: numeral(row["Hospital Beds"]).value(),
```

Comment:
Same question here as above, since we're just indexing straight in with no fallback.